src/pgroonga.c

```diff
@@ -1437,6 +1437,7 @@ PGrnCollectScoreMultiColumnPrimaryKey(Relation table,
 									  HeapTuple tuple,
 									  PGrnScanOpaque so)
 {
+	const char *tag = "pgroonga: [score][multi-column-primary-key][collect]";
```

Comment:
`pgroonga: ` is prepended by `PGrnCheck()` automatically.

```suggestion
	const char *tag = "[score][multi-column-primary-key][collect]";
```